packages/dds/tree/src/shared-tree/schematizeTree.ts

```diff
@@ -73,11 +73,6 @@ export function initializeContent(
 		};
 	}
 
-	// TODO: fix issues with schema comparison and enable this.
-	// assert(
-	// 	allowsRepoSuperset(defaultSchemaPolicy, tree.storedSchema, incrementalSchemaUpdate),
-	// 	"Incremental Schema update should support the existing empty tree",
-	// );
```

Comment:
This assert is passing for the test suite now. I don't know when the issue was fixed. It is being removed since the new assert added to updateSchema is the same as this, except applies in more cases.